.eslintrc

```diff
@@ -0,0 +1,44 @@
+{
+  "env": {
+    "node": true
+  },
+  "globals": {},
+  "rules": {
+    "no-bitwise": 0,
+    "camelcase": 0,
```

Comment:
Is there any reason to mention rules we are not enabling? It seems cleaner to simply omit them.